### DISCUSSION TITLE
GSoC 2022 : JUnit 5 support

### DISCUSSION THREAD

**Discussion by Anish528** [robolectric/robolectric#7153] [General]
Hi Everyone,
I would like to explore this project JUnit 5 Support as part of the GSoC program.
Any advice or tips on how to get started would really help out!
Thanks.

**Comment by utzcoz** [comment]
@Anish528 Welcome to Robolectric, and it's glad that you have interested in project to support Robolectric for JUnit5. It's recommended to follow http://robolectric.org/contributing/ to setup Robolectric development environment locally firstly.  It's also recommended to learn the supporting state of Robolectric from existing issues and related links. Feel free to ask any question about Robolectric and this project.

**Comment by hoisie** [comment]
@Anish528 could you try to follow the instructions at http://robolectric.org/building-robolectric/ and see if you can get to the point where you can build Robolectric? What OS are you using?

**Reply by Anish528** [reply]
Yeah sure I will try and follow the instructions and get back as soon as possible, And I use Windows OS.

**Reply by hoisie** [reply]
If you use windows it might be easier to develop Robolectric in an Ubuntu VM. Not a lot of Robolectric development happens on Windows, so there will be a lot of rough edges. For instance, I'm not sure how easy it isnto use IntelliJ in windows for Robolectric due to the native runtime building. Maybe this is something we should improve.

**Reply by Anish528** [reply]
Ok sure will try building it with Ubuntu VM.
